src/index.ts

```diff
@@ -1,18 +1,14 @@
-/* @flow strict */
-
 const CLOSE_ATTR = 'data-close-dialog'
 const CLOSE_SELECTOR = `[${CLOSE_ATTR}]`
 
-type Focusable =
-  | HTMLButtonElement
-  | HTMLInputElement
-  | HTMLAnchorElement
-  | HTMLTextAreaElement
-  | HTMLSelectElement
-  | HTMLElement
+type Target = Disableable | Focusable
+
+type Disableable = HTMLButtonElement | HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement
+
+type Focusable = HTMLAnchorElement | HTMLElement
```

Comment:
Since `HTMLAnchorElement` extends `HTMLElement` could this just be:

```suggestion
type Focusable = HTMLElement
```

?